src/containers/MoviePage/MoviePage.jsx

```diff
@@ -1,61 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { connect } from 'react-redux';
-
-import { Paper } from '@material-ui/core';
-
-import { PAGE_COUNT } from '../../constants';
 import SearchContainer from '../SearchContainer';
-import FlatPagination from '../../components/FlatPagination';
 import MovieListContainer from '../MovieListContainer';
-import {
-  setPage as setPageAction,
-} from '../../actions/movies';
 
-@connect(
-  state => ({
-    ...state,
-  }),
-  {
-    setPage: setPageAction,
-  },
-)
-export default class MoviePage extends React.PureComponent {
-  changePage(offset) {
-    const { setPage } = this.props;
-    setPage({ page: (offset / PAGE_COUNT + 1) });
-  }
+import Pagination from '../Pagination';
+
+import { PaginationWrapper, MovieWrapper } from './style';
 
+export default class MoviePage extends React.PureComponent {
   render() {
-    const {
-      title, totalResults, isLoading, type, page,
-    } = this.props;
+    const { title, type } = this.props;
     return (
       <div>
-        <FlatPagination
-          onClickPage={(e, offset) => this.changePage(offset)}
-          page={page}
-          totalResults={totalResults}
-        />
-        <Paper>
-          <SearchContainer title={title} />
+        <PaginationWrapper>
+          <Pagination />
```

Comment:
Move business logic from component to the container. I mean action dispatching and so on...